examples/cisco/8000e/8000e.pb.txt

```diff
@@ -18,6 +23,31 @@ nodes: {
         key: 57400
         value: {
             name: "gnmi"
+            outside: 57400
+            inside: 57400
+        }
+    }
+   services:{
```

Comment:
all the comments for the other file apply here as well

Reply:
done